README.md

```diff
@@ -180,8 +180,8 @@ vars:
   - i2_custom_features:
     ApiListener:                #ObjectType
       api:                      #ObjectName
-        accept_command: true    #ObjectAttribute
-        accept_config: true     #ObjectAttribute
+        accept_command:" true"  #ObjectAttribute
```

Comment:
remove empty space after quote accept_command:" true" > accept_command: "true"

Reply:
Please fix that small issue so the code can be merged.